### DISCUSSION TITLE
`as any` in tests

### DISCUSSION THREAD

**Discussion by Aprillion** [mswjs/data#145] [General]
I noticed `as any` assertions in `test/utils/inheritInternalProperties.test.ts` in #135, we could convert similar TS assertions to `@ts-expect-error` for an additional regression test of the types - if the production-code type was changed so that TS no longer produces an error, TS would complain about the missing error in the test code, e.g.:

```diff
 expect(() =>
-  inheritInternalProperties({ firstName: 'John' }, { id: 'abc-123' } as any),
+  // @ts-expect-error
+  inheritInternalProperties({ firstName: 'John' }, { id: 'abc-123' })
 ).toThrow(
```

**Comment by kettanaito** [comment]
Hey, @Aprillion. Thanks for the suggestion.

That TypeScript error suppressed by "any" is not intended and is caused by the second argument that is a corrupted entity (on purpose). 

I think marking it with "expect-error" is precisely what that comment was designed to do. Would you like to open a pull request please?

**Comment by Aprillion** [comment]
PR #146 .. not sure why GitHub didn't link it automatically ¯\\\_(ツ)_/¯

**Reply by kettanaito** [reply]
I guess it doesn't auto-link to discussions. Huh. Thank you for the manual reference 👍 
The changes look superb.
